### PR TITLE
get('/') -> get('/*')

### LINKED ISSUE #104: Server interfere with client router
**Describe the bug**
After running the server and opening the browser on it's port (3000), the server will interfere with the client `router` and the user will get an `Cannot GET /path` error

**To Reproduce**
1. Run server (in server dir `npm start`)
2. Browse to `localhost:3000`
3. Refresh

**Expected behavior**
Normal refresh, any other pat `page-not-found works!`

**Desktop (please complete the following information):**
 - OS: Win 10
 - Browser: Chrome
 - Version 70.0.3538.110

### PR DESCRIPTION
**Pull request by ZimGil**
# Allowing client router

 - Server `get('/')` method changed to `get('/*')` [resolves #104]
   > moved to bottom to allow other methods

### CODE REVIEW

File: server/server.ts [hunk 1]
```diff
@@ -197,6 +193,10 @@ app.listen(PORT, () => {
   }
 });
 
+app.get('/*', (req, res) => {
```
Comment (ortichon) [inline server/server.ts]: why did you change the place of this code block?

Comment (ZimGil) [reply server/server.ts]: If on top, will intercept all other commands.

**Comment by ZimGil** [comment]
This is being resolved at #108
